cmake/Abseil.cmake

```diff
@@ -0,0 +1,22 @@
+include(FetchContent)
+
+set(ABSL_ENABLE_INSTALL ON)
+if (NOT (DEFINED ABSEIL_PATH))
+  message("ABSEIL_PATH not provided. Downloading.")
+  FetchContent_Declare(
```

Comment:
added in 3.11 and we don't require that (to support older linux distributions).

i think you want to follow what the GoogleTest.cmake script does.

Reply:
https://github.com/google/benchmark/issues/603 for the last discussion about bumping cmake versions.

Reply:
Ugh. Lots of cmake wizardry. Got it.